src/HornedBeast.css

```diff
@@ -0,0 +1,11 @@
+article {
+  border: 1px solid black;
+  margin-top: 2vw;
+  padding: 1vw;
+  width: 30%;
+}
+
+article img {
+  width: 100%;
+  margin-bottom: 1vw;
+}
```

Comment:
- We just need to tweak some of the styles to get the app to be responsive
- As screen size shrinks we will need to see a column once it hits a certain width, and then if screen size is large we want to display them properly, side by side , left to right 
- Grab a TA and see if they can help you adjust some of the widths. 👍